client/src/pages/dashboard/views/Staff.svelte

```diff
@@ -52,6 +55,8 @@
         topToastMessage.enqueue({ title: err.name, body: err.message });
         throw err;
     });
+
+    let showUnprev = false;

```

Comment:
The double-negative looks clumsy and bugprone. Let's prefer to use an affirmative naming scheme.